libs/books/data-access/src/lib/+state/reading-list.effects.ts

```diff
@@ -58,5 +70,24 @@ export class ReadingListEffects implements OnInitEffects {
     return ReadingListActions.init();
   }
 
-  constructor(private actions$: Actions, private http: HttpClient) {}
+  openSnackBar(message: string, action: string, undoAction: string, book) {
+    const actionType: string = undoAction;
+    const snack = this._snackBar.open(message, action, {
+      duration: 5000,
+    });
+    snack.onAction().subscribe(()=>{
+      if(actionType==="undoRemove"){
```

Comment:
Watch formatting here. Make sure you use Prettier extension in your IDE.

Reply:
Addressed review comment